src/util/grutil.dsc

```diff
@@ -0,0 +1,21 @@
+grutilbar:
```

Comment:
name the file and script something actually readable. also add `mortal` in front of the script name (for example `mortal_bossbar`)